--- xine-lib-1.2.9/src/libw32dll/wine/Makefile.in.0001~	2018-01-13 02:36:11.576817759 +0100
+++ xine-lib-1.2.9/src/libw32dll/wine/Makefile.in	2018-01-13 02:37:24.736052946 +0100
@@ -576,7 +576,7 @@ MAINTAINERCLEANFILES = Makefile.in
 # disable -fomit-frame-pointer, -finline-functions, and -frename-registers
 # because they cause bad behavior of wine
 AM_CFLAGS = $(DEFAULT_OCFLAGS) $(X_CFLAGS) @W32_NO_OPTIMIZE@ \
-        -fno-omit-frame-pointer -fno-inline-functions -fno-rename-registers
+        -fno-omit-frame-pointer -fno-inline-functions
 
 EXTRA_DIST = stubs.s wrapper.S
 noinst_HEADERS = \
