diff -up xine-lib-1.2.11/src/video_dec/dav1d.c.omv~ xine-lib-1.2.11/src/video_dec/dav1d.c
--- xine-lib-1.2.11/src/video_dec/dav1d.c.omv~	2022-01-12 21:15:10.476788479 +0100
+++ xine-lib-1.2.11/src/video_dec/dav1d.c	2022-01-12 21:17:00.207105809 +0100
@@ -537,11 +537,10 @@ static video_decoder_t *_open_plugin(vid
 
   /* multithreading */
   ncpu = xine_cpu_count();
-  settings.n_frame_threads = (ncpu > 8) ? 4 : (ncpu < 2) ? 1 : ncpu/2;
-  settings.n_tile_threads = MAX(1, ncpu - settings.n_frame_threads + 1);
+  settings.n_threads = (ncpu > 8) ? 4 : (ncpu < 2) ? 1 : ncpu/2;
   xprintf(stream->xine, XINE_VERBOSITY_DEBUG, LOG_MODULE ": "
-            "Using %d frame threads, %d tile threads\n",
-            settings.n_frame_threads, settings.n_tile_threads);
+            "Using %d threads\n",
+            settings.n_threads);
 
   /* dri frame allocator */
   settings.allocator.cookie = this;
